src/test/java/com/artipie/vertx/VertxSliceServerTest.java

```diff
@@ -122,7 +122,7 @@ public void serverHandlesBasicRequest() {
 
     @Test
     public void basicGetRequest() {
-        final String expected = "Hello World!!!";
+        final String expected = "Hello World!!!!";
```

Comment:
@Sammers21 let's keep unrelated test intact, you may use different string constant in new test

Reply:
@Sammers21 please revert changes in this line and we are good to merge